apps/web/src/locales/en/messages.json

```
@@ -5429,6 +5429,37 @@
   "organizationDataOwnership": {
     "message": "Enforce organization data ownership"
   },
+  "organizationDataOwnershipDesc": {
+    "message": "Require all items to be owned by an organization, removing the option to store items at the account level.",
+    "description": "This is the policy description shown in the policy list."
+  },
+  "organizationDataOwnershipContent": {
+    "message": "All items will be owned and saved to the organization, enabling organization-wide controls, visibility, and reporting. When turned on, a default collection be available for each member to store items. Learn more about managing the ",
+    "description": "This will be used as part of a larger sentence, broken up to include links. The full sentence will read 'All items will be owned and saved to the organization, enabling organization-wide controls, visibility, and reporting. When turned on, a default collection be available for each member to store items. Learn more about managing the credential lifecycle.'"
+  },
+  "organizationDataOwnershipContentAnchor":{
+    "message": "credential lifecycle",
+    "description": "This will be used as a hyperlink"
+  },
+  "organizationDataOwnershipWarningTitle":{
+    "message": "Are you sure you want to proceed?"
+  },
+  "organizationDataOwnershipWarning1":{
+    "message": "will remain accessible to members"
+  },
+  "organizationDataOwnershipWarning2":{
+    "message": "will not be automatically selected when creating new items"
+  },
+  "organizationDataOwnershipWarning3":{
+    "message": "cannot be managed from the Admin Console until the user is offboarded"
+  },
+  "organizationDataOwnershipWarningContentTop":{
+    "message": "By turning this policy off, the defualt collection: "
```

Typo I spotted while demoing this:

```suggestion
    "message": "By turning this policy off, the default collection: "
```